cores/esp8266/mmu_iram.h

```diff
@@ -204,13 +245,13 @@ extern void _text_end(void);
 
 static inline __attribute__((always_inline))
 void *mmu_sec_heap(void) {
-  uint32_t sec_heap = (uint32_t)_text_end + 32;
-  return (void *)(sec_heap &= ~7);
+  uintptr_t sec_heap = (uintptr_t)_text_end + (uintptr_t)32u;
+  return (void *)(sec_heap &= ~(uintptr_t)7u);
 }
 
 static inline __attribute__((always_inline))
 size_t mmu_sec_heap_size(void) {
-  return (size_t)0xC000UL - ((size_t)mmu_sec_heap() - 0x40100000UL);
+  return (size_t)0xC000ul - ((uintptr_t)mmu_sec_heap() - (uintptr_t)0x40100000ul);
```

Comment:
I wonder if this was mentioned before, but... isn't it IRAM_START? Could be easier to track named consts instead of just a number?
Also note that the rest of `#define ...` inside of the func blocks above is kind-of misleading, as it is global for anything including this header

Reply:
I found a #define for IRAM start, I'll use it.

For #define inside of a function. I did that as an indication that it was the only place it was being used. I see that was not a clear thing to do. I wanted to use `constexpr`, however, that would not work for `.c` modules. I see that the optimizer to works very well in absorbing const into the code. I'll replace the #define(s) with const instead.